docs/api/rest/agents.md

```
@@ -0,0 +1,295 @@
+---
+sidebar_position: 10
+---
+
+# Agent Management
+
+### List Agents
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)